### PR TITLE
GRPC Apis for Corresponding HTTP APIs

### PR DESCRIPTION
**Pull request by Prashant-Dwivedi-08-01**
Hey @stefanprodan
This PR is in the continuation of our previous effort to include the gRPC APIs for podinfo to extend it's application.
Previous Approved PR:  https://github.com/stefanprodan/podinfo/pull/322
We introduced few APIs last time to test the stability of Podinfo with those. Now we are ready with remaining set of APIs.

Please have a look at this PR that includes the remaining set of APIs.
Thank You

Fix: #318

### CODE REVIEW

**Review by stefanprodan — COMMENTED** [review]
@Prashant-Dwivedi-08-01 can you please document all added functions in the main README under `gRPC API`? Thanks

**Comment by Prashant-Dwivedi-08-01** [comment]
@stefanprodan As per your comment, I have added description of all the gRPC APIs in the main README, you can review it once.
Apart from this, I have a request that if you can look once into why one of the checks is failing in GitHub Actions.
I am getting this error "no language version declared in module.cue" under "Verify CUE formatting".
It asks for adding the language version, but I am not sure why we should edit module.cue file.
Linking it here directly: https://github.com/stefanprodan/podinfo/actions/runs/9631176029/job/26562620984

Please see if you can help. Thanks

**Comment by stefanprodan** [comment]
Please rebase with the master branch, CI should be fixed.

**Comment by Prashant-Dwivedi-08-01** [comment]
@stefanprodan thanks for that recommendation. CI is fixed. You can consider reviewing once and approving the PR. Thanks

File: pkg/api/grpc/status.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	"google.golang.org/grpc/status"
+)
+
+type StatusServer struct {
+	pb.UnimplementedStatusServiceServer
+	config *Config
+	logger *zap.Logger
+}
+
+// SayHello implements helloworld.GreeterServer
```
Comment (stefanprodan) [inline pkg/api/grpc/status.go:18]: Please remove this and the other lines below of commented code

Comment (Prashant-Dwivedi-08-01) [reply pkg/api/grpc/status.go]: Did that! Removed unnecessary comments and whitespaces in code.
Please Check Once. Thanks

**Comment by stefanprodan** [comment]
Can you please run `go fmt ./...` and commit any changes.

**Comment by Prashant-Dwivedi-08-01** [comment]
> Can you please run `go fmt ./...` and commit any changes.

@stefanprodan did that!

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @Prashant-Dwivedi-08-01 🏅

**Pull request merged**